### DISCUSSION TITLE
Does tukaan have sun valley theming?

### DISCUSSION THREAD

**Discussion by not-nef** [tukaan/tukaan#95] [Question]
@rdbende Because in the docs it doesnt look like it

**Comment by Moosems** [comment]
Right now he's adding theming but at the current moment there is no theming. Check [this](https://github.com/tukaan/tukaan/issues/62) thread for more

**Reply by rdbende** [reply]
> at the current moment there is no theming

... at all

However theming will play a big part of Tukaan. TBH, I can't even move forward with other stuff at the moment, until the theming is done. 
Somewhere it will be radically different from Tkinter theming, but still a bit familiar. Actually what is described in #62 is far from everything I have in mind...
